index.d.ts

```diff
@@ -153,6 +153,117 @@ interface Request {
   }
 }
 
+interface ContentOptions {
+  /**
+   * Controls the way the HTMLRewriter treats inserted content.
+   *
+   * - true: Raw HTML
+   * - false: Text and any HTML will be escaped
+   */
+  html: boolean
+}
+
+interface Element {
+  /**
+   *  The namespace URI of the element according to Infra Spec
+   * (https://infra.spec.whatwg.org/#namespaces).
+   */
+  namespaceURI: string
+  /**
+   * e.g. "div"
+   */
+  tagName: string
+  /**
+   * Read-Only - key/value pairs of attributes.
+   */
+  readonly attributes: Iterator<{ name: string; value: string }>
+  /**
+   * Indicates whether the element was removed/replaced in a previous handler
+   */
+  removed: boolean
+
+  getAttribute(name: string): string | null
+  hasAttribute(name: string): boolean
+  setAttribute(name: string, value: string): Element
+  removeAttribute(name: string): Element
+  before(content: string, settings?: ContentOptions): Element
+  after(content: string, settings?: ContentOptions): Element
+  prepend(content: string, settings?: ContentOptions): Element
+  append(content: string, settings?: ContentOptions): Element
+  replace(content: string, settings?: ContentOptions): Element
+  setInnerContent(content: string, settings?: ContentOptions): Element
```

Comment:
Since the type name is `ContentOptions` it makes sense to name the variable `options`